astropy/modeling/tests/test_parameters.py

```diff
@@ -830,14 +830,14 @@ def test_bool(self):
         if param:
             assert True
         else:
-            assert False
+            raise AssertionError()
 
         # vector value at least one false
         param = Parameter(name="test", default=[1, 2, 0, 3, 4])
         assert np.all(param.value == [1, 2, 0, 3, 4])
         assert not np.all(param)
         if param:
-            assert False
+            raise AssertionError()
         else:
             assert True
```

Comment:
This should just be:
```python
assert not param
```
Its testing the `__bool__` method for `Parameter`

Reply:
This should extend to all such cases in `test_parameters.py` which have been updated.

Reply:
Ok. Done!